checkov/terraform/context_parsers/parsers/variable_context_parser.py

```diff
@@ -26,10 +27,16 @@ def get_entity_context_path(self, entity_block: Dict[str, Dict[str, Any]]) -> Li
         return [entity_name]
 
     def enrich_definition_block(self, definition_blocks: List[Dict[str, Any]]) -> Dict[str, Any]:
-        self.context = super().enrich_definition_block(definition_blocks)
-        for variable_block in definition_blocks:
-            if isinstance(variable_block, dict):
-                self._collect_default_variables_values(variable_block)
+        for entity_block in definition_blocks:
+            entity_name, entity_config = next(iter(entity_block.items()))
+            self.context[entity_name] = {
+                "start_line": entity_config[START_LINE],
+                "end_line": entity_config[END_LINE],
+                "code_lines": self.file_lines[entity_config[START_LINE] - 1: entity_config[END_LINE]],
+            }
```

Comment:
@gruebel Is this duplicated from `checkov/terraform/context_parsers/parsers/module_context_parser.py`?

Reply:
they are slightly different